po/de.po

```diff
@@ -2083,15 +2126,15 @@ msgstr "Hilfreiche Fehlermeldungen"
 
 #: data/org.cvfosammmm.Setzer.appdata.xml.in:19
 msgid "Looks great on the Gnome desktop"
-msgstr ""
+msgstr "Sieht gut in Gnome aus"
 
 #: data/org.cvfosammmm.Setzer.appdata.xml.in:20
 msgid "Good screen to content ratio"
-msgstr ""
+msgstr "Hohes Verhältnis von Inhalt zu Fenstergröße"
```

Comment:
Maybe I got this wrong - but shouldn't the english version be the other way around? I.e. high ratio of content compared to the screen size.

Reply:
Low can be "good" in this case. I would say "Gutes Verhältnis ..." in German as well.